aea/cli/registry/utils.py

```diff
@@ -0,0 +1,37 @@
+import click
+import requests
+
+from settings import API_URL
```

Comment:
API_URL replaced `DEFAULT_REGISTRY_PATH`

Reply:
`DEFAULT_REGISTRY_PATH ` leads us to a folder path when `API_URL ` is a web path. A different logic uses those constants (local dir search and search request to API).